public/app/AppWrapper.tsx

```diff
@@ -107,18 +108,18 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
                 >
                   <ModalsProvider>
                     <GlobalStyles />
-                    {config.featureToggles.commandPalette && <CommandPalette />}
+                    {!isPublicDashboardView() && config.featureToggles.commandPalette && <CommandPalette />}
```

Comment:
sorry for bad / short comment, was on the phone :) was not very constructive

Reply:
The purpose of the `isPublicDashboardView()` checks in AppWrapper is to exclude components that aren't relevant to a public viewer, like the nav bar, search, etc. This way, a public dashboard appears like the screenshot in the PR.

I know its not ideal. We will be refactoring this before GA. Are your concerns about the short-term, or long-term?

Reply:
If it is behind a dev feature flag do we need these changes in AppWrapper? Can't they wait then until we have more in place?

Reply:
I think it would feel a bit better if this logic was hidden behind navBarEnabkled(), commandPaletteEnabled(), globalSearchEnabled() functions, and not a !isPublicDashboardView() all over , sometimes in combo with feature toggles